pages/index/index.js

```diff
@@ -3,6 +3,7 @@ import glamorous from 'glamorous'
 import {LiveProvider, LiveEditor, LiveError} from 'react-live'
 import {LiveContextTypes} from 'react-live/lib/components/Live/LiveProvider'
 import Head from 'next/head'
+import {AlgoliaLink, AlgoliaScript} from '../../components/algolia-config'
```

Comment:
Could you put these changes in the layout component? That way it's on all pages. Thanks!